process/peer/process.go

```diff
@@ -525,3 +585,23 @@ func (p *validatorStatistics) IsInterfaceNil() bool {
 	}
 	return false
 }
+
+func (vs *validatorStatistics) getRating(s string) uint32 {
+	peer, err := vs.getPeerAccount([]byte(s))
+
+	if err != nil {
+		log.Debug("Error getting peer account", "error", err)
+	}
+
+	return peer.GetRating()
```

Comment:
this will panic. in case of error return 0 or default.

Reply:
fixed